tests/core/test_datamodules.py

```diff
@@ -312,4 +312,4 @@ def test_full_loop_ddp_spawn(tmpdir):
     # test
     result = trainer.test(datamodule=dm)
     result = result[0]
-    assert result['test_acc'] > 0.8
+    assert result['test_acc'] > 0.7
```

Comment:
@Borda  I think one reason this guy is randomly dropping below the threshold could be because we don't set the deterministic flag in the Trainer.

Reply:
let's just add it to be sure